recipes-fsl/packagegroups/packagegroup-fsl-tools-testapps.bb

```diff
@@ -37,6 +46,24 @@ RDEPENDS_${PN} = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', \
                          'weston-examples clutter-1.0-examples', '', d)} \
     ${SOC_TOOLS_TEST} \
+    ${SOC_TOOLS_TESTAPPS} \
+    procps \
+    ptpd \
+    linuxptp \
+    iw \
+    can-utils \
+    cpufrequtils \
+    nano \
+    ntpdate \
+    minicom \
+    coreutils \
+    mmc-utils \
+    udev-extraconf \
+    e2fsprogs-resize2fs \
+    openssl-bin \
```

Comment:
This does not make sense. We are not trying to provide a full image here but the bare minimum to validate the devices.

This belongs to a custom image but not here.